File: Java/1114 - Senha Fixa.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
         Scanner input =new Scanner(System.in);
         int senha;
             do{
                senha = input.nextInt();
             if (senha == 2002){
                System.out.print("Acesso Permitido\n");
             }else{
                System.out.print("Senha Invalida\n");
             }
            }
             while (senha != 2002);   
 }}

File: C++/1114 - Senha Fixa .cpp
# include <iostream>
# include <stdlib.h>
using namespace std;
int main (){
	int senha;
	cin >>senha;
		do{
		cout << "Senha Invalida" << endl;
		cin >> senha;
		if (senha == 2002){
		cout <<	"Acesso Permitido" << endl;	
		}
		}
		while (not (senha == 2002));
}